File: Starter Problems 1/sit.py
input = open('sitin.txt', 'r')
a, b = map(int, input.readline().split())
c = int(input.readline())

e = 0
if a * b <= c:
	e += a * b
	c -= a * b
else:
	e += c
	c = 0

out = open('sitout.txt', 'w')
out.write(str(e) + ' ' + str(c))

File: Starter Problems 1/sit.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ifstream inFile;
    inFile.open("sitin.txt");
    int r,s,t;
    inFile >> r;
    inFile >> s;
    inFile >> t;
    inFile.close();
    int seats = r*s;
    int standing = 0;
    int sitting = (int)t;
    if (t > seats) {
        standing = t - seats;
        sitting = (int) seats;
    }
    ofstream outFile;
    outFile.open("sitout.txt");
    outFile << sitting << " " << standing;
}
